README.md

```diff
@@ -250,11 +260,11 @@ result in any data loss as the local instances would just queue up their events.
 
 So on your local machine, you'd run:
 
-    oxd --forward=master:3514
+    oxd --forward=log-master
 
 And on the master collection machine, you'd run:
 
-    oxd --collect="*:3514" --log-path=/var/log/blueox/
+    oxd --collect="*" --log-path=/var/log/blueox/
```

Comment:
Actually this will error out since the uid won't have permissions to create this directory. For blueox setup, does it install as root or ask for sudo?